package-lock.json

```diff
@@ -1,332 +1,2508 @@
 {
-  "name": "fraudninja",
+
+    
+
```

Comment:
Nit: lots of blank lines added here now; maybe we can format this (and `package.json`) with deno,prettier, etc? I have this installed already so i can do it if you like

Reply:
Yes good point you noticed it, I also use Prettier but I forgot to Resave the file so that the formatting would be apply. Should be good now